### PR TITLE
Addressbook

### PR DESCRIPTION
**Pull request by mr-tron**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM golang:1.20 as build
```
Comment (aleksej-paschenko) [inline Dockerfile:1]: how about using alpine images? 

```Dockerfile
FROM golang:1.19-alpine AS builder
// build
FROM alpine:latest
// copy
```

File: pkg/addressbook/addressbook.go [hunk 2]
```diff
@@ -14,12 +14,11 @@ import (
...
 type KnownAddress struct {
-	IsScam      bool `json:"is_scam,omitempty"`
-	RequireMemo bool `json:"require_memo,omitempty"`
-	// Name is a dns name.
-	Name    string `json:"name"`
-	Address string `json:"address"`
-	Image   string `json:"image,omitempty"`
+	IsScam      bool   `json:"is_scam,omitempty"`
+	RequireMemo bool   `json:"require_memo,omitempty"`
+	Name        string `json:"name"`
```
Comment (aleksej-paschenko) [inline pkg/addressbook/addressbook.go:19]: it would be super helpful to explain what all these fields are used for, especially `Name`.

**Review by aleksej-paschenko — APPROVED** [review]
looks great!

File: Dockerfile (hunk 1 above)
Comment (mr-tron) [reply Dockerfile:1]: i didn't check by alpine sometimes has problems with C bindings

File: pkg/addressbook/addressbook.go (hunk 2 above)
Comment (mr-tron) [reply pkg/addressbook/addressbook.go:19]: ok.

**Pull request merged**